src/index.js

```diff
@@ -49,6 +52,74 @@ const EmojiPicker = ({
     []
   );
 
+  useEffect(() => {
```

Comment:
I think that all the code that you added to this file begs to be in a separate hook. Something like:

```js
function useKeyboardNavigation() {
	// ...
}
```